### PR TITLE
Fix Goerli testnet link

### PR DESCRIPTION
**Pull request by minimalsm**
## Description
The Prater testnet URL is currently down. The plan is to redirect this to https://goerli.launchpad.ethereum.org/en/. 

- This PR fixes the broken URLs to use the Goerli launchpad
- Slight tweak to copy

## Related Issue
None

### CODE REVIEW

File: src/content/developers/docs/development-networks/index.md
Comment (minimalsm) [inline src/content/developers/docs/development-networks/index.md:65]: Let's not link to /en/ path

```suggestion
- [Goerli Staking Launchpad](https://goerli.launchpad.ethereum.org/)
- [Ropsten Staking Launchpad](https://ropsten.launchpad.ethereum.org/)
```

**Review by corwintines — APPROVED** [review]
LGTM, will let @wackerow reply to comment before merging though.

File: src/intl/en/page-staking.json
Comment (wackerow) [reply src/intl/en/page-staking.json:159]: We had been trying to make the default more the testnet link... We could certainly discuss changing this or updating the surrounding copy. Don't think it needs to block this PR though

**Review by wackerow — APPROVED** [review]
I think this is good to at least fix the links. 
I would say after the Goerli merge is complete, we could go back through and generally remove the usage of "Prater", and add this to deprecation list.

Side thought, makes me wonder if we should expand the "Deprecated software" page to also just include deprecated "names"... like Prater or Eth2 (since the software isn't really deprecated in these cases, just the name... could help for historical understanding)

**Pull request merged**
